README.md

```diff
@@ -1,14 +1,26 @@
-# Ember-spectrum-color-picker
+my-addon
```

Comment:
The main readme header seems to have gotten overwritten with the generator default

Reply:
good catch, i'll fix